src/shared/components/ncTable/partials/rowTypePartials/RowFormWrapper.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<div class="row space-T">
+		<div class="fix-col-4">
+			<div class="title">
+				{{ title }}{{ mandatory ? '*': '' }}
```

Comment:
The * could have a span with a title providing a title that this is a required field maybe? 

Or do we have any other pattern to clarify that fields are required @nimishavijay ?